### PR TITLE
Support for core properties

### PR DESCRIPTION
**Pull request by vrurg**
Read/write.

There is uncertainty about keywords since the implementation is based upon ECMA-376, 5th Edition, but Microsoft Excel for Mac v16.69.1 doesn't seem to support the spec.

### CODE REVIEW

File: lib/Spreadsheet/XLSX.rakumod [hunk 1]
```diff
@@ -54,10 +62,11 @@ class Spreadsheet::XLSX does Spreadsheet::XLSX::Root {
             }
 
             # Locate the root relationships file, and using it, the workbook root.
-            with self.find-relationships('') -> Spreadsheet::XLSX::Relationships $top-rel {
-                with $top-rel
+            with $!root-relationships = self.find-relationships('') {
+                with $!root-relationships
```
Comment (jnthn) [inline lib/Spreadsheet/XLSX.rakumod:66]: Why the nested `with`? Given the assignment on the previous line, it's going to be the same outcome as the outer `with`?

Comment (vrurg) [reply lib/Spreadsheet/XLSX.rakumod:66]: Perhaps an overlooked leftover. Will look into it later today as well as into fixing the conflict.

Comment (vrurg) [reply lib/Spreadsheet/XLSX.rakumod:67]: Ah, no, there is no problem here. The nested `with` is for a chain of method calls `$!root-relationships.find-by-type(...).first`. `find-by-type` argument is a long schema URI, so the chain is split into multiple lines for readability.

**Comment by vrurg** [comment]
I resolved the conflict and made it all clear now.

**Comment by jnthn** [comment]
@vrurg Thanks for the various improvements; I did a release.

**Pull request merged**